dev-libs/granite/granite-6.0.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+BUILD_DIR="${WORKDIR}/${P}-build"
+
+inherit meson vala xdg
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+BDEPEND="
+	$(vala_depend)
+	virtual/pkgconfig
+"
+DEPEND="
+	>=dev-libs/glib-2.50:2
+	>=x11-libs/gtk+-3.22:3[introspection]
+	dev-libs/libgee:0.8[introspection]
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	vala_src_prepare
+}
+
+src_configure() {
+	meson_src_configure
+}
```

Comment:
Could you visibly disable the `documentation` configure option here, while adding a link above it to the upstream issue like I asked?

I also noticed the upstream meson build system has `introspection` option, but I guess we don't need it since nobody's asked for it yet. But `introspection` is quite popular USE flag in Gentoo.